lib/ecl/ecl_grid.cpp

```diff
@@ -3603,6 +3624,21 @@ ecl_grid_type * ecl_grid_alloc(const char * grid_file ) {
 }
 
 
+// This function is used to override use of the keyword ACTNUM from the EGRID file.
+// ext_actnum must have size equal to the number of cells in the main grid
+// if ext_actnum = NULL, actnum is taken from file, otherwise ext_actnums
+// determines which cells are active.
+ecl_grid_type * ecl_grid_alloc_ext_actnum(const char * grid_file, const int * ext_actnum) {
+  ecl_file_enum file_type = ecl_util_get_file_type(grid_file , NULL ,  NULL);
+  if (file_type == ECL_EGRID_FILE)
+    return ecl_grid_alloc_EGRID_all_grids(grid_file, true, ext_actnum);
+  else
+    util_abort("%s must have .EGRID file - %s not recognized \n", __func__ , grid_file);
```

Comment:
Would prefer if your wrote a (empty ?) `ecl_grid_alloc_GRID_all_grids()` which takes the `ext_actnum` - the body of that function can just be `util_abort()`